db/migrate/20190808181844_add_rpm_data_to_amr_counts.rb

```diff
@@ -0,0 +1,9 @@
+class AddRpmDataToAmrCounts < ActiveRecord::Migration[5.1]
+  def change
+    add_column :amr_counts, :annotation_gene, :string
+    add_column :amr_counts, :genbank_accession, :string
+    add_column :amr_counts, :total_reads, :bigint
```

Comment:
Nit: Maybe prefer `:integer` to `:bigint`? I believe `:integer` takes up fewer bytes (4 instead of 8) and still stores very big numbers.